### PR TITLE
Nathan McL

### PR DESCRIPTION
**Pull request by nathanMcL**
in preparation to delete the branch

### CODE REVIEW

**Comment by learner-long-life** [comment]
Good start to merging @nathanMcL looks like you pulled and merged from main, and now you still have some conflicts to resolve before pushing back to `main` and deleting this branch. Keep at it, we need your contributed lines.

File: dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/ArrayWrapper.java [hunk 1]
```diff
@@ -36,17 +50,43 @@ public Iterator iterator() {
...
     public boolean add(Object o) {
-        return false;
+        this._array[currentSize]=o;
+        this.currentSize++;
+        this.isEmpty = false;
+        return true;
     }
 
     @Override
+    //iterate throught the array
```
Comment (faulkdf) [inline dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/ArrayWrapper.java:68]: This set of code could be useful to my final project.

File: dsa-23au/notes/docs/week-01/2023-09-28-Co-Creation.md [hunk 2]
```diff
@@ -2,8 +2,80 @@
...
+
+They will always have an invariant, which is a known stating state.
+ Readjusting work to fit around this class was wise.
+Learning how to work Gitpod/Github is hard.
+I need to get blue light glasses to protect my eyes.
+Bringing water is a great idea so I can focus.
+I consume caffeine in order to stay awake, but it would be better to drink Matcha than Coffee.
+Git is alot of fun
+
+1. I would like more direction from the instructer. I find this horribly challenging not having any steps to guide us.
```
Comment (faulkdf) [inline dsa-23au/notes/docs/week-01/2023-09-28-Co-Creation.md:81]: Im glad to see in the co-creation sentences that we are all in the same boat for the most part in terms of confusion.

File: dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/ArrayWrapper.java [hunk 3]
```diff
@@ -36,17 +50,43 @@ public Iterator iterator() {
...
     public boolean remove(Object o) {
-        return false;
+        int count = null; 
+        for(i = 0; i<=this._array.length; i+=1){
+            if(o == this.array[i]){
+                count = i;
+                break;  
+            }
+        }
+        //Once found, create an array and add all elements except the removed one
```
Comment (faulkdf) [inline dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/ArrayWrapper.java:77]: Comments are very helpful!

**Pull request closed without merging**